drivers/sensor/si7006/Kconfig

```diff
@@ -0,0 +1,11 @@
+#
+# Copyright (c) 2019 Electronut Labs
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+config SI7006
+	bool "Si7006 Temperature and Humidity Sensor"
+	depends on (I2C && HAS_DTS_I2C)
+	help
+	  Enable I2C-based driver for SI7006 Temperature and Humidity Sensor.
```

Comment:
`Si7006`.

Also in binding YAML file (two occurrences).

Reply:
Done.

Reply:
It was done in the YAML, but not here (in the help string).

Reply:
okay, missed it. Corrected now.